docs/FeatureManagement/Clients/SpringCloud/Spring.FeatureFlag.v1.0.0.schema.json

```diff
@@ -0,0 +1,89 @@
+    "$id": "http://azconfig.io/schemas/FeatureManagement/v1.0.0/SpringFeatureFlag.json",
+    "type": "object",
+    "title": "An Spring Cloud Azure App Configuration Feature Declaration",
+    "required": [
+        "feature-management"
+    ],
+    "properties": {
+        "feature-management": {
+            "$id": "#/properties/feature-management",
+            "type": "object",
+            "title": "The Feature Management Schema",
+            "properties": {
+                "patternProperties": {
+                    ".*": {
+                        "anyOf": [
+                            {
+                                "type": "boolean",
+                                "title": "On/Off Feature Flag",
+                                "description": "A feature flag that always returns the same value.",
+                                "examples": [
+                                    true,
+                                    false
+                                ]
+                            },
+                            {
+                                "type": "object",
+                                "title": "Conditional Feature Flag",
+                                "description": "A feature flag which value is dynamic based on a set of feature filters",
+                                "default": null,
+                                "properties": {
+                                    "enabled-for": {
+                                        "$id": "#/properties/feature-management/properties/.*/properties/EnabledFor",
+                                        "type": "array",
+                                        "title": "The declaration of conditions used to dynamically enable features.",
+                                        "default": null,
+                                        "items": {
+                                            "$id": "#/properties/feature-management/properties/.*/properties/EnabledFor/items",
+                                            "type": "object",
+                                            "title": "The filter information associated with the conditional.",
+                                            "required": [
+                                                "name",
+                                                "parameters"
+                                            ],
+                                            "properties": {
+                                                "name": {
+                                                    "$id": "#/properties/feature-management/properties/.*/properties/EnabledFor/items/properties/Name",
+                                                    "type": "string",
+                                                    "title": "Client Filter Name",
+                                                    "description": "The name used to refer to and require a client filter.",
```

Comment:
Is client filter the terminology used in Spring Cloud feature management? In .NET Core we use "feature filter"

Reply:
It should be "feature filter". There should be no differences in terms of terminology between languages.